src/PockyBot.NET/Services/Triggers/Rotation.cs

```diff
@@ -28,7 +28,7 @@ public async Task<Message> Respond(Message message)
 
             foreach (var item in rotation.Split(','))
             {
-                builder.Append($"* {item}\n");
+                builder.AppendLine($"* {item}");
```

Comment:
This uses environment specific newlines so it's different depending on the OS you run it on. So the tests will fail on Windows